### DISCUSSION TITLE
See a blank screen when using with npm module with version 1.3; whereas, works okay when using unpack in script tags.

### DISCUSSION THREAD

**Discussion by MImranAsghar** [google/model-viewer#1944] [Q&A]
### Description

Please provide a detailed description of the bug, how to reproduce it, and the
expected behavior.  Always include a code snippet, screenshot, any errors
reported in the console, and the model to help us understand and fix the
problem.

#### Live Demo

https://glitch.com/edit/#!/model-viewer

Example of issue when using npm module with react
https://codesandbox.io/s/gifted-elgamal-r3crj?file=/src/App.js
### Version

- model-viewer: vX.X.X

### Browser Affected

- [v ] Chrome
- [ ] Edge
- [ v] Firefox
- [ ] IE
- [v ] Safari

### OS

- [ ] Android
- [ ] iOS
- [ ] Linux
- [ v] MacOS
- [v ] Windows

**Comment by elalish** [comment]
Can you please fill out the rest of the bug template, especially the repro link? I'll need to be able to repro it to understand the problem.

**Comment by MImranAsghar** [comment]
Hi @elalish! just added an example on template

**Comment by elalish** [comment]
Okay, so this is a React issue? You may want to search our issues for React, as there have been a number of questions asked and answered. I'm no React specialist, but I know many of our users have it working.
